src/main/java/net/irisshaders/iris/api/v0/item/IrisItemColorProvider.java

```diff
@@ -0,0 +1,14 @@
+package net.irisshaders.iris.api.v0.item;
+
+import net.coderbot.iris.vendored.joml.Vector3f;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+
+public interface IrisItemColorProvider
+{
```

Comment:
Why is the `{` on a separate line, this isn't csharp lmao

Reply:
oops, IDE settings